learning/courses/quantum-business-foundations/introduction-to-quantum-computing.ipynb

```diff
@@ -0,0 +1,308 @@
+    "| <em> 1996 </em> | A year later, Lov Grover, also a Bell Labs scientist, came up with a quantum algorithm that would allow people to swiftly search unstructured databases. Scientists piled into the field, and advances in hardware soon followed the breakthroughs in code. [2]|\n",
+    "| <em> 1998 </em> | The first experimental demonstration of a quantum algorithm was achieved in 1998. A working 2–qubit nuclear magnetic resonance (NMR) quantum computer was used to solve Deutsch’s problem by Jonathan A. Jones and Michele Mosca at Oxford University and shortly after by Isaac L. Chuang at IBM’s Almaden Research Center and Mark Kubinec and the University of California, Berkeley, together with coworkers at Stanford University and MIT. [3]|\n",
+    "| <em> 2001 </em> | 2001 saw the first execution of Shor’s algorithm at IBM’s Almaden Research Center and Stanford University. The number 15 was factored using 1018 identical molecules, each containing seven active nuclear spins. [4]|\n",
+    "| <em> 2005 </em> | By the mid–2000s, the research field had developed several types of superconducting qubits, each with its own advantages and drawbacks. In 2007, a team at Yale found a way to combine some of these approaches to overcome their individual shortcomings, naming the new design the “transmon qubit.” The transmon qubit would go on to be at the heart of many companies’ efforts to develop quantum computers, including IBM Quantum, Google AI, and Rigetti Computing. A member of the Yale team, Jay Gambetta, later became Vice President of Quantum Computing for IBM Research. |\n",
+    "\n",
+    "![IBM’s four-qubit quantum computer](/learning/images/courses/quantum-business-foundations/introduction-to-quantum-computing/fourqubitdevice_small.avif)\n",
+    "\n",
+    "*Layout of IBM’s four-qubit superconducting quantum computer announced in 2015. (Credit: IBM Research)*\n",
+    "\n",
+    "\n",
+    "|         |       |\n",
+    "| ----------------- | -- |\n",
+    "| <em> 2016 </em> | In May of 2016, IBM was the first company to launch a cloud quantum computing service that included real quantum computers, called IBM Quantum Experience. [5] |\n",
+    "\n",
+    "![The launch of the IBM Quantum Experience in 2016 enabled users to access quantum computers from anywhere, even their tablet.](/learning/images/courses/quantum-business-foundations/introduction-to-quantum-computing/IBM-Quantum-Experience_small.avif)\n",
+    "\n",
+    "*IBM Quantum Composer on a tablet at IBM Research (Credit: Connie Zhou for IBM)*\n",
+    "\n",
+    "|         |       |\n",
+    "| ----------------- | -- |\n",
+    "| <em> 2017 </em> | In March of 2017, IBM released Qiskit, an open-source quantum programming framework. [6] In December of 2017, the IBM Quantum Network was launched to establish a quantum computing commercial ecosystem.|\n",
```

Comment:
All the references need to be links to `(#references)`. See an example:

https://github.com/Qiskit/documentation/blob/851377a495d15eed1108eb36ec4df5360d0c0fe9/docs/tutorials/wire-cutting.ipynb#L22 

https://github.com/Qiskit/documentation/blob/851377a495d15eed1108eb36ec4df5360d0c0fe9/docs/tutorials/wire-cutting.ipynb#L1233-L1246